ntbs-build/openshift-pipelines/pipelines/run-load-tests.yml

```diff
@@ -0,0 +1,91 @@
+apiVersion: tekton.dev/v1beta1
+kind: Pipeline
+metadata:
+  name: run-load-tests
+  namespace: ntbs-build
+spec:
+  params:
+    - default: live
+      description: The version of the load tests to run
+      name: branch
+      type: string
+    - default: 'https://ntbs-uat.phe.nhs.uk'
+      description: The url of the site under test
+      name: url
+      type: string
+    - description: The authenticated token used to make requests to the site under test
+      name: token
+      type: string
+    - default: '30'
+      description: The length of the test in minutes
+      name: length_in_minutes
+      type: string
+    - default: '329414'
```

Comment:
any reason for this number?

Reply:
The notifications on UAT at present start at 329414